### DISCUSSION TITLE
No detection between devices

### DISCUSSION THREAD

**Discussion by dasarkisov** [schlagmichdoch/PairDrop#356] [General]
Hi! I'm first time to use PairDrop (official instance), and just can't get my devices to detect each other.
I have:

- Android device with installed "Snapdrop & PairDrop" app from Google Play; connected via WiFi.
- Windows 11 device with PairDrop desktop app installed from pairdrop.net via Edge Browser; connected via LAN.
- Both devices are in the same network (WiFi/LAN) and are paired.

Still, there is no detection.

I've tried different ways to connect (android: app/chrome-web + windows: app/edge-web). Every once in a while I manage to get them to see each other and the file transfer works., but it happens very rarely and it is unclear under what conditions.

**Comment by schlagmichdoch** [comment]
Do the browser console logs show any error message? It would help if you could post them here.
Also, if you open two PairDrop tabs on the same device do they show each other?

**Reply by dasarkisov** [reply]
> Do the browser console logs show any error message? It would help if you could post them here. Also, if you open two PairDrop tabs on the same device do they show each other?

As far I found out, here are the combinations that do work:
- desktop app + desktop browser (same PC);
- desktop app + desktop app (remote paired PC);
- desktop browser + desktop browser (remote paired PC);
- destop browser + desktop app (remote paired PC);
- desktop app + desktop browser (remote paired PC);
(Though, as fas as I concerned, browser and PC-app verisons are all the same)

Android (app/browser) still won't properly work in any combination.

Which exactly browser console logs shoud I check? The Android's one, as it is the one to fail?

**Reply by schlagmichdoch** [reply]
What Android or Android browser version do you use?

**Reply by dasarkisov** [reply]
> What Android or Android browser version do you use?

I use Android 14 and the up to date Google Chrome browser (v. 131.0.6778.200). The app is either the "Snapdrop & PairDrop" (latest version from Google Chrome), or the Chrome-installed web-app.
